pkg/auth/token_verifier.go

```diff
@@ -255,3 +267,8 @@ type openIDMetadata struct {
 	SubjectTypes  []string `json:"subject_types_supported"`
 	SigningAlgs   []string `json:"id_token_signing_alg_values_supported"`
 }
+
+type filtersBySubjects struct {
+	filters  []eventingv1.SubscriptionsAPIFilter
+	subjects []string
+}
```

Comment:
Wonding, if "subjectsWithFilters" or "filtersForSubjects" somehow would be a better name here

Reply:
yeah probably, I was having a hard time coming up with a good name for this. I'll use "subjectsWithFilters" - thanks!